frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -26,6 +27,22 @@ const CourseAttribute = ({ title, content }) => {
   );
 };
 
+const ProgressBar = ({progress,height}) => {
```

Comment:
This component should be seperated to another file under the `components` folder with its own js and css/less files included.